### DISCUSSION TITLE
Excel FILTER Formula on Table Excluding Headers

### DISCUSSION THREAD

**Discussion by giscus** [elibroftw/blog.elijahlopez.ca#12] [General]
# Excel FILTER Formula on Table Excluding Headers

Enter a variant of the formula below where Table1 is the name of the table (Ctrl + T to create tables in Excel). Since we are using a formula in another sheet, it might be prudent to place this on row 3 to make room for custom headers on row 1 and custom data (or another filter) on row 2.
# Enter this in A5 =FILTER(Table1[Stock]:Table1[Status],NOT(ISBLANK(Table1[Status]))) # Enter this in C5 or the width of the first formula + 1 =FILTER(Table1[Speculation]&"",NOT(ISBLANK(Table1[Status]))) The reason there are two formulas is because empty cells show up as “0” by filter, so if we need spill-overs, we need to use the &"" modifier on each array separately.

http://localhost:1313/posts/excel-filter-table-excluding-headers/

**Comment by elibroftw** [comment]
Test giscus
